README.md

```diff
@@ -170,12 +170,15 @@ Make sure to follow [CONTRIBUTING guidelines](./CONTRIBUTING.md#Adding-and-Updat
 
 <a name="building-with-gitpod"></a>
 
-## Building with Gitpod 💣
+## Building with Gitpod | Codespaces 💣
 
-By using [Gitpod. io](https://www.gitpod.io), all the necessary dependencies will be installed
+By using [Gitpod. io](https://www.gitpod.io), [Codespaces](https://github.com/codespaces) all the necessary dependencies will be installed
```

Comment:
[proofreading] This off to a great start. It explains how to use these two online IDEs. However, new contributors would probably wonder why they should consider using it. I suggest briefly explaining that in the beginning like in the example below: 
```suggestion
Not comfortable with using a local IDE yet or just want to make a quick edit? No worries, you can always use [Gitpod. io](https://www.gitpod.io) or [Codespaces](https://github.com/codespaces). They install all the necessary dependencies
```

Reply:
Want to avoid the complexities of setting up a local environment or just need to make quick edits? Consider using [Gitpod.io](https://www.gitpod.io/) or [Codespaces](https://github.com/codespaces). They streamline the process by automatically installing required dependencies, making it effortless to contribute in just one click
How is this